src/main/java/ro/msg/learning/shop/CSV2JSON/WebConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package ro.msg.learning.shop.CSV2JSON;
+
+        import org.springframework.context.annotation.Configuration;
```

Comment:
Format all your code please (right click on the ro.msg.learning.shop package -> Reformat Code).